components/chart/index.tsx

```diff
@@ -55,24 +58,36 @@ function Chart({
     ZOOM_OUT: () => changeZoomLevel(10),
   };
 
+  const toggleDrawer = () => {
+    if (description !== '') {
+      setDescription('');
+    }
+  };
+  const findModuleById = (id: number): Record<string, string> | undefined => {
+    const arrayOfNodes = Object.keys(currentVersion.nodes).map((key) => currentVersion.nodes[key]);
+    const module: Record<string, string> = arrayOfNodes.find((module) => module.id === id);
+    return module ? module : undefined;
+  };
+
   const selectModule = useCallback(
-    (id: number, groupSelect?: boolean) => {
+    (module: NodeType, groupSelect?: boolean) => {
+      setSelectedModule(module);
+      setDescription(module.description);
       let newSelectedModules = selectedModules;
       if (groupSelect) {
         if (
           newSelectedModules &&
-          newSelectedModules.filter((m) => m === id).length <= 0
+          newSelectedModules.filter((m) => m === module).length <= 0
```

Comment:
I think here it's wrong
```suggestion
          newSelectedModules.filter((m) => m === module.id).length <= 0
```